### PR TITLE
surface bugs with a lot of cc & votes

### PR DESCRIPTION
**Pull request by sylvestre**

### CODE REVIEW

File: auto_nag/scripts/lot_of_votes.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        return bug
+
+    def get_bz_params(self, date):
+        params = {
+            "include_fields": ["creation_time", "last_change_time"],
+            "resolution": "---",
+            "f1": "days_elapsed",
+            "o1": "lessthan",
+            "v1": self.nweeks * 7,
+#            "f2": "cc_count",
```
Comment (calixteman) [inline auto_nag/scripts/lot_of_votes.py:38]: useless ?

File: auto_nag/scripts/lot_of_votes.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            "votes_type": "greaterthaneq",
+            "votes": "10",
+            "f2": "votes",
+            "o2": "greaterthaneq",
```
Comment (calixteman) [inline auto_nag/scripts/lot_of_votes.py:42]: `v2` is missing

**Comment by calixteman** [comment]
Usual questions:
 - did you dry-run the tools ?
 - did you get the expected results ?

**Comment by marco-c** [comment]
Might make sense to merge all these tools into a single one (find bugs with lots of duplicates OR lots of cc OR lots of votes OR lots of see also) and generate a single report  / propose increasing severity for all of them.

File: auto_nag/scripts/lot_of_votes.py (hunk 2 above)
Comment (sylvestre) [reply auto_nag/scripts/lot_of_votes.py:42]: Surprisingly, it isn't :) 
it is using votes as a replacement

File: auto_nag/scripts/lot_of_votes.py (hunk 1 above)
Comment (sylvestre) [reply auto_nag/scripts/lot_of_votes.py]: fixed

**Comment by sylvestre** [comment]
>  did you dry-run the tools ?
Yes

>  did you get the expected results ?
yes

**Comment by sylvestre** [comment]
> Might make sense to merge all these tools into a single one (find bugs with lots of duplicates OR lots of cc OR lots of votes OR lots of see also) and generate a single report / propose increasing severity for all of them.

I would prefer to keep them separated as they don't say the same thing :)

**Comment by marco-c** [comment]
> I would prefer to keep them separated as they don't say the same thing :)

We can still show the difference by listing # of dups, CC, votes in separate columns and have different thresholds for each, but at least we can share the code between them (especially when we will implement the action to propose increasing severity).
We can do that in a follow-up though.

**Comment by sylvestre** [comment]
yeah, next step :)

**Review by calixteman — APPROVED** [review]
LGTM.

**Pull request merged**